presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -197,29 +197,16 @@ private PlanBuilder appendInPredicateApplyNode(PlanBuilder subPlan, InPredicate
 
         checkState(inPredicate.getValueList() instanceof SubqueryExpression);
         PlanNode subquery = createRelationPlan(((SubqueryExpression) inPredicate.getValueList()).getQuery()).getRoot();
-        Map<Expression, Symbol> correlation = extractCorrelation(subPlan, subquery);
-        if (!correlationAllowed && correlation.isEmpty()) {
-            throwNotSupportedException(inPredicate, "Correlated subquery in given context");
-        }
-        subPlan = subPlan.appendProjections(correlation.keySet(), symbolAllocator, idAllocator);
-        subquery = replaceExpressionsWithSymbols(subquery, correlation);
 
-        TranslationMap translationMap = subPlan.copyTranslations();
```

Comment:
Changes in this class does not match the commit message, probably they need to be done in separate commit.